packages/effect/src/Effect.ts

```diff
@@ -13013,6 +13013,9 @@ export const Tag: <const Id extends string>(id: Id) => <
     return makeTagProxy(TagClass as any)
   }
 
+/** @internal */
+type MissingSelfGeneric = `Missing \`Self\` generic - use \`class Self extends Service<Self>()...\``
```

Comment:
```suggestion
type MissingSelfGeneric = `Missing \`Self\` generic - use \`class Self extends Effect.Service<Self>()...\``
```

I think including the Effect namespace in the message is clearer.